zcutil/fetch-params.sh

```diff
@@ -27,8 +24,18 @@ function fetch_params {
             --continue \
             "$url"
 
-        # Only after successful download do we update the parameter load path:
-        mv -v "$dlname" "$output"
+        shasum -a 256 --check <<EOF
+$expectedhash  $dlname
+EOF
+
+        # Check the exit code of the shasum command:
+        CHECKSUM_RESULT=$?
+        if [ $CHECKSUM_RESULT -eq 0 ]; then
```

Comment:
Does this need to be quoted, or is `$?` guaranteed to never return an empty string?